elastic/kibana
# [Ingest Manager] Wording & linking improvements

## Summary

Fixes https://github.com/elastic/kibana/issues/75807 .

### How to test this

Compare changes with the linked issue.

To verify the link to license management, start with only a basic license, e.g. with `yarn es snapshot --license basic [...]` instead of `yarn es snapshot --license trial [...]`
